example/index.html

```diff
@@ -7,7 +7,23 @@
   </head>
   <body>
     <p>Press F12 to open the console</p>
-    <canvas id="qrCanvas"></canvas>
+    <p><b>Waku Status:</b> <span id="waku-status">connecting...</span></p>
+    <p id="handshake-span"><b>Handshake Status:</b> <span id="handshake-status">-</span></p>
+    <canvas id="qr-canvas"></canvas>
+    <a href="#" id="qr-url" style="display: none" target="_blank">Click here to simulate QR code being scanned</a>
```

Comment:
```suggestion
    <a href="#" id="qr-url" style="display: none" target="_blank">Open QR code link in new window instead of scanning it.</a>
```

Trying to make it a bit clearer.